wiki/Article_Style_Guide/en.md

```diff
@@ -1,30 +1,57 @@
 # Article Style Guide
 
+This Article Style Guide applies to all articles in English and other languages that does not have one yet!
+
 The Article Style Guide serves as a way to help keep consistency in clarity, formatting, and layout between all articles of the osu!wiki.
 
 All English articles should use plain English.
-Your word choice should be able to explain such things in layman's terms (basically ask yourself, "If the reader is reading this, would they have to lookup any of the words?").
+Your word choice should explain the topic that the article aims to explain thoroughly in layman's terms.
+In other words, you should ask yourself, "If the reader is reading this, would they have to lookup any of the words?"
 
 All articles must have proper grammar, correct spelling, and correct information.
-Know that reviewers may ask for changes in your pull request for blunders or suggestions.
-A good osu!wiki writer should read these reviews to help improve the overall quality of these articles to ensure optimal experience for an osu!wiki reader.
+Keep in mind that reviewers would ask for changes in your pull request for blunders or suggestions.
+A good osu!wiki writer/editor should read these reviews to help improve the overall quality of these articles to ensure optimal experience for the reader.
 
 ## GFM
 
 GFM (GitHub Flavored Markdown) is the supported markup language used in the osu!wiki.
-HTML use is **heavily** discouraged (and might have your pull request be denied)!!
+Usage of HTML is **heavily** discouraged!
 
-GFM has a small amount of markup that **anyone** can learn it in less than a day!
-However, don't expect a lot out of Markdown as it is intended to be used as a bare minimum markup language.
+GFM has a small amount of markup that **anyone** can learn it in about 3 mintues!
+However, do not expect a lot out of Markdown as it is intended to be used as a bare minimum markup language.
 
 ### Syntax
 
 If you need help with GFM syntax, see [markdown-cheatsheet-online.pdf](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf) or [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
 
-## File names
+If you really need to, you could also ask for some help in the [osu!dev discord](https://discord.gg/ppy).
+
+## Folder Names
+
+All folders must **never** contain URL reserved characters.
+
+Generally, all folder names should only use these characters:
+
+- A - Z (uppercase letters)
+- a - z (lowercase letters)
+- 0 - 9 (numbers 0 - 9)
+- `_` (underscore)
+- `!` (exclaimation point)
+
+### Index Pages
+
+All folders **must** contain a page of some kind, even if they are _index_ pages (pages that link to other pages).
+
+Index pages **must** only link to their own subfolders.
+
+Index pages **must** contain descriptions of what they link to.
+
+## File Names
+
+All articles in the `osu-wiki` repository uses the ISO 639-1 codes.
```

Comment:
I already have :p, I am just waiting for RaikireHiuduo's response before pushing.